pkg/util/pravegacluster_test.go

```diff
@@ -0,0 +1,262 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package util
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestPravegacluster(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Pravegacluster")
+}
+
+var _ = Describe("pravegacluster", func() {
+
+	Context("IsVersionBelow07", func() {
+		var result1, result2 bool
+		BeforeEach(func() {
+
+			result1 = IsVersionBelow07("0.7.1")
+			result2 = IsVersionBelow07("0.6.0")
+			IsVersionBelow07("666")
+			IsVersionBelow07("")
```

Comment:
The result returned by these 2 is not stored or checked

Reply:
done